odoo/addons/base/models/ir_qweb.py

```diff
@@ -594,6 +644,8 @@ def _compile(self, template):
 
             where ``qweb`` is a QWeb instance and ``values`` are the values to
             render.
+
+            :returns tuple containing code, options and main method name
```

Comment:
rST info fields start and end with a `:`.

This should be `:returns: <whatever>`